app/build.gradle.kts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2022 usuiat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.jetbrains.kotlin.gradle.dsl.KotlinJvmOptions
+import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSetTree
+
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.multiplatform)
+    alias(libs.plugins.jetbrains.compose)
+}
+
+kotlin {
+    androidTarget()
+    jvm("desktop")
+
+    targets.all {
+        compilations.all {
+            kotlinOptions.let {
+                if (it is KotlinJvmOptions) {
+                    it.jvmTarget = "1.8"
+                }
+            }
+        }
+    }
+
+    applyDefaultHierarchyTemplate {
+        sourceSetTrees(KotlinSourceSetTree.main, KotlinSourceSetTree.test)
+        common {
+            group("nonAndroid") {
+                withJvm()
+            }
+        }
+    }
```

Comment:
I don't see why we need nonAndroid group.
Currently nonAndroidMain has platformSamples, but I think `desktopMain` can have it.

The same goes for zoomable/nonAndroidTest

Reply:
Prepare for Js，wasmjs，iOS target，although those target not supported yet，

Reply:
> I don't see why we need nonAndroid group.
> Currently nonAndroidMain has platformSamples, but I think `desktopMain` can have it.
> 
> The same goes for zoomable/nonAndroidTest 

Accompanist library is a Android library ，other platform(exclude Android) don't need this list ，so we using the `Android` and `nonAndroid`

Reply:
Okey.

I understand that we write Android code in androidMain.
But I don't think it is a good way to creat a `nonAndroid` to write non-Android code.
This way, we will need a `nonIos` when writing iOS code, and a `nonJs` when writing js code, and we'll end up with a mess.

However, I don't know what the best way is at the moment.
Please understand that I may change the sourceset hierarchy in the future.